source/includes/_dataset_security.md

```diff
@@ -0,0 +1,339 @@
+`is_data_visible` <br> *boolean* | Flag indicating whether the target will have access the dataset's records or not.
+`filter_query` <br> *string* | The target will only have access to the records matching this query. An empty query means that all records are accessible.
+`permissions` <br> *array of strings* | List of special permissions granted to the target. <br> Only available for user and group-level rulesets. <br> Possible values are `edit_dataset`, `publish_dataset`, `manage_dataset`
+`api_calls_quota` <br> *[quota object](#the-quota-object)* | Upper limit set on the number of api calls the target can make to this dataset in a given timeframe. <br> Can be set to null for no specific quota.
+`user` <br> *[user object](#the-user-object)* <br> <em class="expandable">expandable</em> | The user targeted by this ruleset. <br> Only available for user-level rulesets.
+`group` <br> *[group object](#the-group-object)* <br> <em class="expandable">expandable</em> | The group targeted by this ruleset. <br> Only available for group-lebel rulesets.
+
+<aside>
+    Setting <code>is_data_visible</code> to <code>false</code> will void the effect of the <code>filter_query</code> value. No data at all will be visible by the target.
+</aside>
+
+## Retrieve the default security ruleset
+
+Retrieves the default security ruleset, that is the ruleset that applies when no specific ruleset is declared for the user or one of their groups.
+
+> Definition
+
+```http
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/security/default
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXXX/security/default \
+    -u username:password
+```
+
+> Example response
+
+```json
+{
+    "is_data_visible": false,
+    "visible_fields": ["field1", "field2"],
+    "filter_query": "",
+    "api_calls_quota": null,
+    "permissions": []
+}
+```
+
+## Update the default security ruleset
+
+PUT `datasets/<dataset_uid>/security/default`
+
+## Reset the default security ruleset
+
+DELETE `datasets/<dataset_uid>/security/default`
+
+## Retrieve all user level security rulesets
+
+Retrieves all rulesets defined for specific users. The rulesets are ordered by ascending username.
+
+> Definition
+
+```http
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/security/users
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXXX/security/users \
+    -u username:password
+```
+
+> Example response
+
+```json
+[
+    {
+        "user": {
+            "username": "username"
+        },
+        "is_data_visible": false,
+        "visible_fields": ["field1", "field2"],
+        "filter_query": "",
+        "api_calls_quota": null,
+        "permissions": []
+    },
+    {...},
+    {...}
+]
+```
+
+## Create a new user level security ruleset
+
+POST `datasets/<dataset_uid>/security/users/`
```

Comment:
First occurrence of a trailing slash in this page. I prefer to have them but if we're not having them we need to stay consistent with it.